demos/controlgroup/toolbar.html

```diff
@@ -0,0 +1,257 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Controlgroup - Toolbar</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/button.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/menu.js"></script>
+	<script src="../../ui/position.js"></script>
+	<script src="../../ui/selectmenu.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		var iframe = $( "<iframe id='display' contenteditable='true'>" ),
+			contents = iframe.appendTo( "body" );
+
+		// Firefox and IE require us to wait for next tick before interacting with the iframe
+		setTimeout(function(){
+
+			contents = contents.contents()
+			contents[ 0 ].designMode = "On";
+			contents[ 0 ].contenteditable = true;
+			contents.find( "body" ).append( $( "#input" ).clone().attr( "id", "output" ) );
+
+			$( "#print" ).button({
+				"icon": "ui-icon-print",
+				"showLabel": false
+			});
+			$( "#redo" ).button({
+				"icon": "ui-icon-arrowreturnthick-1-e",
+				"showLabel": false
+			});
+			$( "#undo" ).button({
+				"icon": "ui-icon-arrowreturnthick-1-w",
+				"showLabel": false
+			});
+			$( "#print, #bold, #italic, #underline, #undo, #redo" ).on( "click", function() {
+				contents[ 0 ].execCommand( this.id );
+			});
+			$( "#fontsize, #forecolor, #hilitecolor, #backcolor, #fontname" ).on( "change selectmenuchange", function() {
+				contents[ 0 ].execCommand( this.id ,false, $( this ).val() );
+			});
+			$( ".toolbar" ).controlgroup();
+			$( "#zoom" ).on( "selectmenuchange", function() {
+				contents.find( "#output" ).css({ "zoom": $( this ).val() });
+			})
+			$( "form" ).on( "submit", function( event ) {
+				return false;
+			});
+		});
+	});
+	</script>
+	<style>
+		#zoom, #fontsize {
+			min-width:75px;
+		}
+		#input {
+			display: none;
+		}
+		#input, #display {
+			width: 98%;
+			height: 300px;
+			border: 2px inset #ccc;
+			font-size: 11px;
+			font-family: "Lucida Grande";
+			zoom: 100%;
+			padding: 5px;
+		}
+	</style>
+</head>
+<body>
+<div class="demo-description">
+<p>A sample editor toolbar</p>
+<p>Highlight text and edit it using the buttons and dropdowns in the toolbar</p>
+</div>
+	<div class="toolbar">
+			<button id="print">Print</button>
+			<button id="undo">Undo</button>
+			<button id="redo">Redo</button>
+			<select id="zoom">
+				<option>50%</option>
+				<option>75%</option>
+				<option>90%</option>
+				<option selected>100%</option>
```

Comment:
This also happens for the default demo. There's not even a title attribute, so without clicking its impossible to read the selected option.